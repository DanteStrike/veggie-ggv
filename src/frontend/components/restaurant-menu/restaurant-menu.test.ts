import RestaurantMenu from "./restaurant-menu";

it(`Restaurant Menu component should render correctly`, () => {
  const component = new RestaurantMenu();
  expect(component.getElement()).toMatchSnapshot();
});
