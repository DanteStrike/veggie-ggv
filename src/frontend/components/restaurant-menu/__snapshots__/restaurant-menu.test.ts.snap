// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Restaurant Menu component should render correctly 1`] = `
<div
  class="container"
>
  
            
  <h2
    class="title restaurant-menu__title"
  >
    Menu
  </h2>
  
            
  <nav
    class="nav restaurant-menu__nav"
  />
  
            
  <div
    class="restaurant-menu__offers clearfix"
  />
  
        
</div>
`;
